### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by mKomorek**

### CODE REVIEW

**Comment by mKomorek** [comment]
please read trello card

**Comment by starmarek** [comment]
Ok, my bad. I have updated .env file. Now there is only one warnin: 
Screenshot from 2020-05-02 15-48-35
Please fix it.

File: client/src/toolset/dictionary.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  };
+  
+  this.viewAll = function() {
+    for (var key in this.data) {
+      console.log(key + " -> " + this.data[key]);
+    }
+  };
+}
+
+export default Dictionary;
```
Comment (jaolejnik) [inline client/src/toolset/dictionary.js:59]: Next time remember to add a newline at the end of the file. [Here's why.](https://evanhahn.com/newline-necessary-at-the-end-of-javascript-files/)

File: client/src/App.js [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
-import TemplateDashboard from "./components/template/templateDashboard.component";
-import CreateInviteTemplate from "./components/template/createInviteTemplate.component";
 import SendInvite from "./components/invitation/sendInvite.component";
-import CreateUser from "./components/user/createUser.component";
 import UserProfile from "./components/user/userProfile.component";
-import Home from "./components/layout/home.component";
+import TemplateDashboard from "./components/views/templateDashboard.component";
+import CreateInviteTemplate from "./components/template/createInviteTemplate.component";
 
+// TODO make it a local variable or something
```
Comment (starmarek) [inline client/src/App.js:12]: Is it possible to remove comment, and use .env varaiables?

Comment (mKomorek) [reply client/src/App.js:12]: No, this is client side

**Pull request merged**